tests/processor/test_tasks.py

```diff
@@ -192,9 +221,9 @@ def setUp(self):
         self.fake_get_report_args = {'customer_name': self.fake.word(),
                                      'authentication': self.fake_account,
                                      'provider_type': 'AWS',
-                                     'report_name': self.fake.word(),
                                      'schema_name': self.fake.word(),
-                                     'billing_source': self.fake.word()}
+                                     'billing_source': self.fake.word(),
+                                     'provider_uuid': '6e212746-484a-40cd-bba0-09a19d132d64'}
```

Comment:
If the uuid doesn't matter for the test, this could easily be `uuid.uuid4()`

Reply:
ah yeah.  So I'm relying on the test account (`test.sql`) here rather than populating the database in the test case.  Not ideal, but for this purpose seemed ok to me.